blazor/scheduler/resources.md

```diff
@@ -442,6 +445,8 @@ To get start quickly about multiple resource on scheduler, you can check on this
     }
 }
 ```
+The following image depicts how appointments belonging to the different resources will be displayed altogether on the default Scheduler.
```

Comment:
The following image shows the appointment rendering on the multiple resource Scheduler.

Reply:
The requested changes are made.